backend/app.js

```diff
@@ -1,39 +1,37 @@
-const port = process.env.PORT || 3000,
-    http = require('http'),
-    fs = require('fs'),
-    html = fs.readFileSync('index.html');
+const express = require("express");
+const bodyParser = require("body-parser");
+const fs = require("fs");
+const html = fs.readFileSync("backend/index.html");
 
-const log = function(entry) {
-    fs.appendFileSync('/tmp/sample-app.log', new Date().toISOString() + ' - ' + entry + '\n');
-};
-
-const server = http.createServer(function (req, res) {
-    if (req.method === 'POST') {
-        let body = '';
+const app = express();
+const port = process.env.PORT || 3000;
 
-        req.on('data', function(chunk) {
-            body += chunk;
-        });
+const log = function (entry) {
+  fs.appendFileSync(
+    "/tmp/sample-app.log",
+    new Date().toISOString() + " - " + entry + "\n"
+  );
+};
 
-        req.on('end', function() {
-            if (req.url === '/') {
-                log('Received a message.');
-            } else if (req.url = '/scheduled') {
-                log('Received task ' + req.headers['x-aws-sqsd-taskname'] + ' scheduled at ' + req.headers['x-aws-sqsd-scheduled-at']);
-            }
+// Middleware to parse JSON bodies
+app.use(bodyParser.json());
 
-            res.writeHead(200, 'OK', {'Content-Type': 'text/plain'});
-            res.end();
-        });
-    } else {
-        res.writeHead(200);
-        res.write(html);
-        res.end();
-    }
+// Endpoint for serving HTML content
+app.get("/", (req, res) => {
```

Comment:
Since we're separating the BE and FE into their own apps/deployments, this is not necessary.

That being said, I think we leave it here, so we can easily test if the server is up... for the time being